src/Infrastructure/DistGrid/interface/ESMF_DistGrid.F90

```diff
@@ -63,7 +63,7 @@ module ESMF_DistGridMod
   sequence
 #endif
   private
-    type(ESMF_Pointer) :: this
+    type(ESMF_Pointer) :: thisq
```

Comment:
Looks like an accidental 'q' made it to the end of this line.

Reply:
Wondering how that didn't cause more problems...

Reply:
It does if you tried to build DistGrid after the 'q' came in. But now its fixed again.